src/encoded/schemas/raw_matrix_file.json

```diff
@@ -0,0 +1,384 @@
+                "title": "Feature key",
+                "type": "string",
+                "enum": [
+                    "Ensembl gene ID",
+                    "gene symbol",
+                    "antibody target",
+                    "genomic coordinates",
+                    "motif ID"
+                ]
+            }
+        },
+        "feature_counts": {
+            "title": "Feature counts",
+            "description": "For each category of feature type, the number of entries found in the matrix considered as features (eg: genes, antibody capture, or peaks)",
+            "comment":  "Do not submit. This will be filled in during checkfiles validation.",
+            "type": "array",
+            "uniqueItems": true,
+            "items": {
+                "title": "Feature count",
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                    "feature_type": {
+                        "title": "Feature type",
+                        "comment":  "Do not submit. This will be filled in during checkfiles validation.",
+                        "type": "string",
+                        "enum": [
+                            "gene",
+                            "peak",
+                            "transcription factor",
+                            "antibody capture"
+                        ]
+                    },
+                    "feature_count": {
+                        "title": "Feature count",
+                        "comment":  "Do not submit. This will be filled in during checkfiles validation.",
+                        "description": "",
+                        "type": "integer"
+                    }
+                }
+            }
+        },
+        "value_scale": {
+            "title": "Value scale",
+            "description": "The factor by which the expression values have been scaled; linear if not scaled.",
+            "type": "string",
+            "default": "linear"
+        },
+        "value_units": {
+            "title": "Value units",
+            "description": "The units for the expression values.",
+            "type": "string",
+            "enum": [
+                "UQ",
+                "SF",
+                "RPM",
+                "RPKM",
+                "FPKM",
+                "TPM",
+                "TMM",
+                "UMI",
+                "expected count",
+                "fragment ends"
+            ]
+        },
+        "normalized": {
+            "title": "Normalized",
+            "description": "A flag to indicate whether the expression values have been normalized.",
+            "type": "boolean",
+            "default": false
+        },
+        "scaled": {
+            "title": "Scaled",
+            "description": "A flag to indicate whether the expression values have been scaled.",
+            "type": "boolean",
+            "default": false
+        },
+        "filtering_cutoffs": {
```

Comment:
Do we need filtering_cutoffs for RawMatrix?

Reply:
Nope. Removed that field from RawMatrixFile